autoconfigure-storage/src/main/java/com/google/cloud/trace/zipkin/autoconfigure/ZipkinStackdriverStorageAutoConfiguration.java

```diff
@@ -45,14 +48,21 @@
 @ConditionalOnProperty(name = "zipkin.storage.type", havingValue = "stackdriver")
 @ConditionalOnMissingBean(StorageComponent.class)
 public class ZipkinStackdriverStorageAutoConfiguration {
+  private static final Logger log = LoggerFactory.getLogger(ZipkinStackdriverStorageAutoConfiguration.class);
+
   @Autowired
   ZipkinStackdriverStorageProperties storageProperties;
 
-  @Bean
+  @Bean(name = "stackdriverExecutor")
   @ConditionalOnMissingBean(Executor.class) Executor executor() {
     ThreadPoolTaskExecutor executor = new ThreadPoolTaskExecutor();
     executor.setThreadNamePrefix("ZipkinStackdriverStorage-");
+    executor.setCorePoolSize(storageProperties.getExecutor().getCorePoolSize());
+    executor.setMaxPoolSize(storageProperties.getExecutor().getMaxPoolSize());
+    executor.setQueueCapacity(storageProperties.getExecutor().getQueueCapacity());
     executor.initialize();
+
+    log.info("Configured Executor for ZipkinStackDriver Storage with: {}", storageProperties.getExecutor());
```

Comment:
nit I'd use jul since this is not advanced and it generally dodges a topic of which logging system to use.